# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmConstants.h
# Commit: 538920d2d777
# Full Hash: 538920d2d77751743a51f10cf730f1ce65b5d0ee
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2020-08-18 09:24:52
# Regressor Bug: 1561521
# File Overlap Count: 2
# Description:
#   Bug 1561521 - Rename TypeCode/ValType/RefType::Any to Extern. r=lth
#   
#   This commit renames the TypeCode/ValType/RefType occurrences of 'any' to 'extern'
#   to match what they actually represent. All public API facing occurrences have
#   already been switched to 'externref' so this should only be aligning the
# ==============================================================================

diff -r 3d3e3d124767 -r 538920d2d777 js/src/wasm/WasmConstants.h
--- a/js/src/wasm/WasmConstants.h	Tue Aug 18 00:03:29 2020 +0000
+++ b/js/src/wasm/WasmConstants.h	Tue Aug 18 00:04:37 2020 +0000
@@ -62,9 +62,8 @@
   // A function pointer with any signature
   FuncRef = 0x70,  // SLEB128(-0x10)
 
-  // A reference to any host type. This is exposed to JS as 'externref' and
-  // will eventually be renamed in code as well.
-  AnyRef = 0x6f,  // SLEB128(-0x11)
+  // A reference to any host value.
+  ExternRef = 0x6f,  // SLEB128(-0x11)
 
   // Type constructor for reference types.
   OptRef = 0x6c,